Keas.Core/Data/ApplicationDbContext.cs

```diff
@@ -52,6 +52,7 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
         public virtual DbSet<GroupXTeam> GroupXTeams { get; set; }
 
         public virtual DbSet<ExtendedPersonView> ExtendedPersonViews { get; set; }
+        public virtual DbSet<AttributeKey> AttributeKeys { get; set; }

```

Comment:
@jSylvestre I'd think `EqiupmentAttributeKeys` table, since this is a list of all keys that will be used in the `EquipmentAttributes` table